LoopKitTests/Fixtures/InsulinKit/suspend_dose_reconciled_normalized.json

```diff
@@ -3,7 +3,7 @@
         "type": "TempBasal",
         "start_at": "2015-07-13T12:00:00",
         "end_at": "2015-07-13T12:00:00",
-        "amount": -0.75,
+        "amount": 0,
```

Comment:
Curious why this change is needed?

Reply:
It's related to how "reconciled and normalized" dose volumes are tracked in the new system: BasalRelativeDose now instead of a DoseEntry that has a scheduled basal rate added to it.  BasalRelativeDose uses *volume* to track total dose amount, not switching between *volume* and *rate* like DoseEntry does. The dose in question has a duration of 0, which means volume is 0.  This is the way suspend events are tracked. At some point we get a resume, and then that becomes a real dose.

This test is maybe ineffectual; it's a little confusing as to what it's even supposed to test; the resulting IOB array at the end of the test is all 0's because the suspend is 0 length; i.e. no insulin effect at all. Looking back in history, it was this way even back to Loop main, before any algo work. 

I think the test is still operating as it does on main, even with this change to `amount`. It would probably make more sense to update the test to assume the suspend had been running for some time, and then have some negative IOB in the final array.